src/components/filter/FilterPanel.tsx

```diff
@@ -130,17 +130,44 @@ const FilterPanel = (props: Props) => {
 			</div>
 			<ModalPortal isOpen={showFilter}>
 				<div ref={filterBaseRef} className='filter'>
-					{config.map(rowConfig =>
-						Array.isArray(rowConfig) ? (
-							<div className='filter__compound' key={rowConfig.map(c => c.id).join('-')}>
-								{rowConfig.map(_rowConfig => (
-									<FilterRow rowConfig={_rowConfig} key={_rowConfig.id} />
-								))}
-							</div>
-						) : (
-							<FilterRow rowConfig={rowConfig} key={rowConfig.id} />
-						),
-					)}
+					{config
+						.sort((a, b) => {
+							const priority = [
```

Comment:
can we move sorting to EventsFilterPanel and MessagesFilterPanel components? it'll be easier to sort it there and make more sense i think. also you can declare this order before component is defined, since it's constant, and we don't need to define it every time inside .sort compare function